frontend/src/redux/configSlice.ts

```diff
@@ -9,6 +9,17 @@ export interface ConfigState {
   clusters: {
     [clusterName: string]: Cluster;
   } | null;
+  /**
+   * Statelss Clusters is a map of cluster names to cluster objects.
```

Comment:
Can you please explain here why they are they stateless here? I don't get the difference between these and clusters.

Reply:
Indeed, while the two types of clusters share many similarities, it's important to distinguish them as 'stateless clusters' for future considerations. Stateless clusters are handled primarily on the frontend, using Redux for retrieval and processing, while normal clusters are fetched from the backend proxy cache. This separation allows us to implement specific features or functionality for stateless clusters in a more streamlined manner.